doc/changes/README.rst

```diff
@@ -0,0 +1,24 @@
+Recording Changes
+=================
+
+This directory contains "news fragments" which are small files containing text that will be integrated into release notes.
+The files can be in restructured text format or plain text.
```

Comment:
Does it support Markdown? Some people are more comfortable with Markdown than restructured text.

Reply:
Yes, you can just do `DM-1234.feature.md` and it will do the right thing (I had an example of that in the daf_butler fragments).